Mathlib/Algebra/Star/StarAlgHom.lean

```diff
@@ -303,35 +297,16 @@ structure StarAlgHom (R A B : Type*) [CommSemiring R] [Semiring A] [Algebra R A]
 by forgetting the interaction with the star operation. -/
 add_decl_doc StarAlgHom.toAlgHom
 
-/-- `StarAlgHomClass F R A B` states that `F` is a type of ⋆-algebra homomorphisms.
-
-You should also extend this typeclass when you extend `StarAlgHom`. -/
-class StarAlgHomClass (F : Type*) (R A B : outParam Type*)
```

Comment:
I wonder if we should keep this around but mark it `@[deprecated]`?

Reply:
done